<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Результат олимпиады: Фильтрация</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>trophy.png</normaloff>trophy.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="school_filter">
       <item>
        <property name="text">
         <string>Все</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="class_filter">
       <item>
        <property name="text">
         <string>Все</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="submit_button">
       <property name="text">
        <string>Узнать результаты</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
